lib/zendesk-uploader/client-wrapper.js

```diff
@@ -68,7 +72,11 @@ module.exports = class ZendeskClientWrapper {
                     // err.retryAfter is present on all 429 (rate limit has been exceeded) and some 503 errors
                     // err.retryAfter is time to wait in seconds
                     retries--;
-                    let retryAfter = err.retryAfter ? err.retryAfter * 1000 : RETRY_AFTER_DEFAULT;
+                    retryNumber++;
+                    const retryAfter = err.retryAfter
+                        ? err.retryAfter * 1000
+                        // Default retry should wait twice as long as the previous retry
+                        : RETRY_AFTER_DEFAULT * Math.max(Math.pow(2, retryNumber - 1), 1);
```

Comment:
Why do we subtract 1 from `retryNumber`?
Also, isn't retry time grow to fast? For 6th retry delay would be 16 seconds. Maybe we need to add also max waiting time?